addon/components/power-select.js

```diff
@@ -337,7 +337,14 @@ export default Ember.Component.extend({
       if (searchAction) {
         this._performSearch(searchAction, term);
       } else {
-        this.setProperties({ results: this.filter(this.get('options'), term), searchText: term, lastSearchedText: term });
+        let options = this.get('options');
```

Comment:
Totally right. I actually did the same in lines 110~116, but not here. This PR just needs tests to prevent a regression, probably in the `general-behaviour-test.js` file.